spork/screens/splicer_screen.py

```diff
@@ -41,18 +66,57 @@ def quit_game(game_state):
     game_state.update({'screen_done': True})
     return game_state
 
-def add_1(game_state):
-    splice_sprites.add(ImageSprite(490, 363, game_state.get('active_sprite1')))
+def dud(game_state): #holding function for buttons that don't do anything yet.
+    print("BOO")
+    return game_state
+
+def toggle_delete_mode(game_state):
+    delete_mode = game_state.get('delete_mode')
+    if delete_mode == False:
+        game_state.update({'delete_mode': True})
+        return game_state
+    if delete_mode == True:
+        game_state.update({'delete_mode': False})
+        return game_state
```

Comment:
Noted. As there is will be some additional complexity to this function I will keep it as it is for now and see if it can be simplified after some other bits have gone into it.